pkg/network/ebpf/c/protocols/classification/routing-helpers.h

```diff
@@ -0,0 +1,81 @@
+#ifndef __PROTOCOL_ROUTING_HELPERS_H
+#define __PROTOCOL_ROUTING_HELPERS_H
+
+#include "ktypes.h"
+#include "protocols/classification/defs.h"
+
+static __always_inline bool is_last_program_from_layer(classification_prog_t current_program) {
```

Comment:
(together with the next comment)
suggest to rename to `has_available_program(program)`

Reply:
That's much better! Will update it 👍

Reply:
Actually, the function does the very opposite of that 😅

Reply:
I ended up changing the function implementation a little bit and sticking to the name you suggested